### PR TITLE
Add subscriptions argument to Subscriber.import

### PR DESCRIPTION
**Pull request by tcollier**
@alan-vb

### CODE REVIEW

**Comment by eremzeit** [comment]
l2gm, though, because we are changing the arity of the function, it could potentially break things.  this is fine.  i'm just making note.

**Comment by jakeonrails** [comment]
@tcollier Did you just merge this without any further commits? There's a merge conflict and a `.idea` directory still lurking in here! :(

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+yesmail2
```
Comment (jakeonrails) [inline .idea/.name:1]: i recommend .gitignoring .idea and `git rm`ing this from the repo

File: yesmail2.gemspec.orig [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name        = 'yesmail2'
+<<<<<<< HEAD
+  s.version     = '0.0.3'
+=======
+  s.version     = '0.0.4'
+>>>>>>> Add subscriptions argument to Subscriber.import
```
Comment (jakeonrails) [inline yesmail2.gemspec.orig:7]: probably want to fix this.

**Comment by jakeonrails** [comment]
Hmm I looked at the master branch and it looks like these issues were corrected. So, NVM! Good work catching that.

**Pull request merged**